Adapt homepage and dependencies for upcoming PhD course (#18)
neuroinformatics-unit/course-animals-in-motion: book/index.qmd

## Preface {.unnumbered}

> **Animals in Motion** is an online handbook designed for researchers and students interested in learning about free open-source tools for tracking animal motion from video footage and extracting quantitative descriptions of behaviour from motion tracks.

Reviewer: ```suggestion
**Animals in Motion** is an online handbook designed for researchers and students interested in learning about free, open-source tools for tracking animal motion from video footage and extracting quantitative descriptions of behaviour from motion tracks.
```

> **Animals in Motion** is an online handbook designed for researchers and students interested in learning about free open-source tools for tracking animal motion from video footage and extracting quantitative descriptions of behaviour from motion tracks.

Reviewer: ```suggestion
**Animals in Motion** is an online handbook that introduces free open-source tools for tracking animal motion from videos and extracting quantitative descriptions of behaviour from motion tracks.
```

broader audience?

Author: Love this! It also makes the sentence much shorter.

> @sec-versions of this handbook have been taught as hands-on workshops in the following settings:

Reviewer: ```suggestion
[Earlier versions @sec-versions] of this handbook have been taught as hands-on workshops in the following settings:
```

- the [Neuroinformatics Unit Open Software Summer School](https://neuroinformatics.dev/open-software-summer-school/index.html)
- the Sainsbury Wellcome Centre's (SWC) [Systems Neurscience PhD Programme](https://www.sainsburywellcome.org/web/content/systems-neuroscience-phd-programme).

Whether you are attending a workshop in person, or following along the materials at your own pace, be sure to check out @sec-prerequisites.

> All materials are shared under the [CC-BY-4.0 license](https://github.com/neuroinformatics-unit/course-animals-in-motion/blob/main/LICENSE.md) so feel free to use them for your own workshops.

Reviewer: ```suggestion
All materials are shared under the [CC-BY-4.0 license](https://github.com/neuroinformatics-unit/course-animals-in-motion/blob/main/LICENSE.md) so you are welcome to use and adapt them for your own workshops.
```

> You don't need to ask for permission, but we'd love to hear about it!

Reviewer: ```suggestion
There is no need to ask for permission, but we'd love to hear about it if you do!
```

> We are also always happy to receive feedback and contributions to the handbook, see @sec-contributing.

Reviewer: ```suggestion
Feedback and contributions to the handbook are always welcome, see @sec-contributing.
```

## Schedule

> In the workshop taught on **October 9th, 2025** to students of SWC's [Systems Neurscience PhD Programme](https://www.sainsburywellcome.org/web/content/systems-neuroscience-phd-programme) we are covering the following chapters of this handbook:

Reviewer: ```suggestion
The workshop on **October 9th, 2025**, for students of the [SWC Systems Neurscience PhD Programme](https://www.sainsburywellcome.org/web/content/systems-neuroscience-phd-programme) will cover the following chapters of this handbook:
```

| Version | Description |
|---------|-------------|
| [dev](https://animals-in-motion.neuroinformatics.dev/dev/) | development version, corresponding to the `main` branch |
| [v2025.08](https://animals-in-motion.neuroinformatics.dev/v2025.08/) | version used during the inaugural [Open Software Week](https://neuroinformatics.dev/open-software-summer-school/2025/index.html) in August 2025 |
| [v2025.08-answers](https://animals-in-motion.neuroinformatics.dev/v2025.08-answers/) | same as `v2025.08` but with answers to exercises |

## Funding & acknowledgements

The inaugural [Open Software Week](https://neuroinformatics.dev/open-software-summer-school/2025/index.html),

> which served as an impetus for creating this handbook, was made possible by a [Software Sustainability Institute](https://www.software.ac.uk/) fellowship to **Niko Sirmpilatze**, as well as further funding support by the [Sainsbury Wellcome Centre](https://www.sainsburywellcome.org/), the [Society for Research Software Engineering](https://society-rse.org/) and [AIBIO-UK](https://aibio.ac.uk/).

Reviewer: ```suggestion
which inspired the creation of this handbook, was made possible by a [Software Sustainability Institute](https://www.software.ac.uk/) fellowship to **Niko Sirmpilatze**, as well as further funding support by the [Sainsbury Wellcome Centre](https://www.sainsburywellcome.org/), the [Society for Research Software Engineering](https://society-rse.org/) and [AIBIO-UK](https://aibio.ac.uk/).
```
